src/scm/bitbucket.js

```diff
@@ -62,131 +64,136 @@ class Bitbucket {
-        {
-          token: this.accessToken,
-          items: [], 
-          urls: [],
-          url: `${this.baseUrl}/repositories/${context.repo.fullName}/src/${response.target.hash}/?access_token=${this.accessToken}`
-        }),
-        this.followDirectory,
-        'bitbucket'
-      )
+      .then(accessToken => {
+        return $.getJSON(
+          `${this.baseUrl}/repositories/${context.repo.fullName}/refs/branches/${context.branch}`, {
+            access_token: accessToken
+          }
+        )
+      })
       .then(response => {
-        const promises = response.map(src => {
-          return new Promise((resolve, reject) => {
-            $.get(src.links.self.href, { access_token: this.accessToken })
-            .then(content => {
-              resolve({ file: src.path, content: content});
-            })
-            .fail(reject)
+        return getAllItems(Promise.resolve({
+              token: this.accessToken,
+              items: [],
+              urls: [],
+              url: `${this.baseUrl}/repositories/${context.repo.fullName}/src/${response.target.hash}/?access_token=${this.accessToken}`
+            }),
+            this.followDirectory,
+            'bitbucket'
+          )
+          .then(response => {
+            const promises = response.map(src => {
+              return new Promise((resolve, reject) => {
+                $.get(src.links.self.href, {
+                    access_token: this.accessToken
+                  })
+                  .then(content => {
+                    resolve({
+                      file: src.path,
+                      content: content
+                    });
+                  })
+                  .fail(reject)
+              });
+            });
+            return Promise.all(promises);
           });
-        });
-        return Promise.all(promises);
       });
-    });
   }
 
   getNamespaces() {
     return this.getAccessToken()
-    .then(accessToken => {
-      return getAllItems(Promise.resolve(
-        {
-          token: accessToken,
-          items: [], 
-          url: `${this.baseUrl}/teams?access_token=${accessToken}&role=contributor`
-        }),
-        this.followPaginate,
-        'bitbucket'
-      );
-    })
-    .then(teams => {
-      this.namespaces = [this.user].concat(teams.map(team => team.username));
-      return this.namespaces;
-    })
-    .catch((err) => {
-      showAlert('Failed to get user info.', LEVEL_ERROR);
-    });
+      .then(accessToken => {
+        return getAllItems(Promise.resolve({
+            token: accessToken,
+            items: [],
+            url: `${this.baseUrl}/teams?access_token=${accessToken}&role=contributor`
+          }),
+          this.followPaginate,
+          'bitbucket'
+        );
+      })
+      .then(teams => {
+        this.namespaces = [this.user].concat(teams.map(team => team.username));
+        return this.namespaces;
+      })
+      .catch((err) => {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/15f4157afeba0560f3f12968d583a6f4fceda1cf/src/scm/bitbucket.js#L171"></span>‘err’ is defined but never used.</td>
    </tr>
  </tbody>
</table>